e2e-tests/gef/cypress/e2e/facility/review-ukef-approval/review-ukef-approval-with-issued-and-unissued-facilities-MIA-and-resubmit-to-ukef.spec.js

```diff
@@ -39,7 +39,7 @@ context('Review UKEF decision MIA -> confirm coverStartDate and issue unissued f
         // creates application and inserts facilities and changes status
         cy.apiCreateApplication(BANK1_MAKER1, token).then(({ body }) => {
           dealId = body._id;
-          cy.apiUpdateApplication(dealId, token, MOCK_APPLICATION_MIA_DRAFT);
```

Comment:
This rename reflects the status that was previously being used here (the status was 'submitted to ukef' as opposed to 'draft' even though it had 'draft' in the title)